packages/medusa/src/services/claim.ts

```diff
@@ -166,7 +166,7 @@ export default class ClaimService extends TransactionBaseService {
             } else {
               await shippingOptionServiceTx.createShippingMethod(
                 method.option_id as string,
-                (method as any).data,
+                {},
```

Comment:
question: Should we allow data on updates?

Could be: 
```
(method as any)?.data ?? {}
```

Reply:
Ah wait. I see that we do not allow the `data` prop at all in the API layer. Fine to keep PR as is, and then add later. But if you think adding `data` to update + create is a minor addition, we might as well go ahead with it and do:
```
(method as any)?.data ?? {}
```

...in both update and create claim.

Reply:
I can add it real quick, but do we also want to add it for returns while we are at it? @olivermrbl